railties/lib/rails/secrets.rb

```diff
@@ -42,7 +42,6 @@ def key
 
       def template
         <<-end_of_template.strip_heredoc
-          # See `secrets.yml` for tips on generating suitable keys.
```

Comment:
We should probably rather link to a new section in the security guide on guides.rubyonrails.org.

Or distribute the info in `rails credentials:tips` or similar.